vnext/Microsoft.ReactNative/ReactPointerEventArgs.idl

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "DocString.h"
+#include "NamespaceRedirect.h"
+
+namespace Microsoft.ReactNative {
+  [webhosthidden]
+  DOC_STRING("Event arguments wrapper for IViewManagerWithPointerEvents.")
+  runtimeclass ReactPointerEventArgs {
+    DOC_STRING("Prevent event from being emitted to React JS component.")
+    XAML_NAMESPACE.Input.PointerRoutedEventArgs Args {
+      get;
+    };
+    DOC_STRING("Gets or sets the React target for the pointer event.")
+    Object Target {
+      get;
+      set;
+    };
+    DOC_STRING("Prevent event from being emitted to React JS component.")
+    void PreventDefault();
+    DOC_STRING("Prevent or release pointer capture in React Native pointer event handler.")
+    void ReleaseCapture();
+    DOC_STRING("Prevent parent views from receiving pointer event callbacks.")
+    void StopPropagation();
```

Comment:
these read more as properties / SetPreventDefault

Reply:
Was trying to keep the names more similar to the Web Event API variants (stopPropagation() and preventDefault()) than following C++ / Windows specific guidelines, but open to changing the API if you have a better suggestion.

Reply:
No longer relevant in this PR. We'll reintroduce SetDefault in a future PR.